packages/SwingSet/src/kernel/kernel.js

```diff
@@ -283,6 +285,36 @@ export default function buildKernel(kernelEndowments, kernelOptions = {}) {
     }
   }
 
+  function removeVatManager(vatID, reason) {
```

Comment:
let's make this `async`, if `shutdown()` somehow throws a synchronous error, I don't want `terminateVat` to do the same